ddtrace/internal/_unpatched.py

```diff
@@ -0,0 +1,10 @@
+# Acquire a reference to the open function from the builtins module. This is
+# necessary to ensure that the open function can be used unpatched when required.
+from builtins import open as unpatched_open  # noqa
```

Comment:
do we only need `from builtins import open` here? we don't have to rename, right?

Reply:
no we don't, but it's kind of nice to be forced to import a different name as a reminder this is the unpatched version.

Reply:
I wonder if we should do the same for threading?

Reply:
This may be done in another refactor? As you want.

Reply:
yeah 👍🏻